src/org/zaproxy/zap/extension/AllInOneNotes/ExtensionAllInOneNotes.java

```diff
@@ -0,0 +1,211 @@
+        // manually, the components added through the class ExtensionHook (in hook(ExtensionHook))
+        // are automatically removed by the base unload() method.
+        // If you use/add other components through other methods you might need to free/remove them
+        // here (if the extension declares that can be unloaded, see above method).
+    }
+
+    private AbstractPanel getStatusPanel() {
+
+        ExtensionHistory extHist = (ExtensionHistory) org.parosproxy.paros.control.Control.getSingleton().
+                getExtensionLoader().getExtension(ExtensionHistory.NAME);
+
+
+        ArrayList<String[]> notes = new ArrayList<>();
+
+        if (extHist != null) {
+
+            int LastHistoryId = extHist.getLastHistoryId();
+
+            int i=0;
+            while (i++ < LastHistoryId) {
+                HistoryReference hr = extHist.getHistoryReference(i);
+                if (hr != null) {
+                    if (hr.hasNote()) {
+                        try {
+                            String note = hr.getHttpMessage().getNote();
+                            String[] tableRow = {String.valueOf(i), note};
+                            notes.add(tableRow);
+                        } catch (HttpMalformedHeaderException e) {
+                        } catch (DatabaseException e) {
+                            e.printStackTrace();
+                        }
+                    }
+                }
+            }
+        }
+
+
+        JXTable notesTable = new JXTable(new NotesTableModel(notes));
+        notesTable.setColumnSelectionAllowed(false);
+        notesTable.setCellSelectionEnabled(false);
+        notesTable.setRowSelectionAllowed(true);
+        notesTable.setAutoCreateRowSorter(true);
+        notesTable.setColumnControlVisible(true);
+
+        exportButton = new TableExportButton<>(notesTable);
+
+        tableSelectionModel = notesTable.getSelectionModel();
+        tableSelectionModel.addListSelectionListener(new NotesTableSelectionHandler(notesTable, extHist));
+
+        JPanel buttonContainer = new JPanel();
+        buttonContainer.setLayout(new BoxLayout(buttonContainer, BoxLayout.X_AXIS));
+        buttonContainer.add(reload);
+        buttonContainer.add(exportButton);
+
+        JPanel pContainer = new JPanel();
+        pContainer.setLayout(new BoxLayout(pContainer, BoxLayout.Y_AXIS));
+        pContainer.add(buttonContainer);
+        pContainer.add(new JScrollPane(notesTable));
+
+        if (statusPanel != null) {
+            statusPanel.removeAll();
+            statusPanel.add(pContainer);
+        } else {
+
+            statusPanel = new AbstractPanel();
+            statusPanel.setLayout(new CardLayout());
+            statusPanel.setName(Constant.messages.getString(PREFIX + ".panel.title"));
+            statusPanel.setIcon(ICON);
+
+            statusPanel.add(pContainer);
+        }
+
+
+        return statusPanel;
+    }
+
+    private ZapMenuItem getMenuExample() {
```

Comment:
It's not Example anymore :grin: (throughout the function)

Reply:
agreed! changed